### PR TITLE
[9.0][MIG] medical patient disease allergy

### PR DESCRIPTION
**Pull request by lasley**
This is dependent on #72

medical_patient_disease_allergy isolates the allergy concept from medical_patient_disease 

- [x] #72

### CODE REVIEW

**Comment by woodbrettm** [comment]
Ready for review when medical_patient_disease merged!

**Comment by lasley** [comment]
@bmw95 - please rebase now that disease is merged

**Comment by lasley** [comment]
@bmw95 - nvm I was in a rebase frenzy and decided to just do these

**Review by lasley — COMMENTED** [review]
LGTM thanks @bmw95 - @t3ddftw please review

File: medical_patient_disease_allergy/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Patient Allergies",
+    "summary": "Isolates allergies from diseases.",
+    "version": "9.0.1.0.0",
+    "category": "Medical",
+    "website": "https://laslabs.com/",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "maintainer": "LasLabs, Odoo Community Association (OCA)",
```
Comment (woodbrettm) [inline medical_patient_disease_allergy/__openerp__.py]: Remove

**Review by woodbrettm — COMMENTED** [review]
LGTM just need to remove maintainer attribute from manifest

**Comment by woodbrettm** [comment]
@lasley @t3ddftw I reviewed again and fixed up some stuff. Looks ready to go.

File: medical_patient_disease_allergy/demo/medical_patient_demo.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    <record id="res_partner_patient_5" model="res.partner">
+        <field name="name">Michael Marine</field>
+        <field name="city">Las Vegas</field>
+        <field name="state_id" ref="base.state_us_23" />
+        <field name="country_id" ref="base.us" />
+        <field name="zip">89521</field>
+        <field name="street">123 Test 2 Drive</field>
+        <field name="email">michael@example.com</field>
+        <field name="phone">+1 (702) 3542-2447</field>
```
Comment (tedsalmon) [inline medical_patient_disease_allergy/demo/medical_patient_demo.xml]: I don't really like the idea of having a phone number that isn't formatted correctly for the country. Maybe we could stick to +1 (702) 555-xxxx for demo data?

**Review by tedsalmon — APPROVED** [review]
Looks good, minus a minor nit-pick in demo data.

File: medical_patient_disease_allergy/demo/medical_patient_demo.xml (hunk 2 above)
Comment (woodbrettm) [reply medical_patient_disease_allergy/demo/medical_patient_demo.xml]: oh woops. Must have accidentally added a 4th num in there.

**Comment by woodbrettm** [comment]
Updated :)

**Review by lasley — COMMENTED** [review]
Thanks @bmw95

**Pull request merged**